multimodal/src/autogluon/multimodal/utils/misc.py

```diff
@@ -45,7 +45,27 @@ def tensor_to_ndarray(tensor: torch.Tensor):
     return tensor.detach().cpu().float().numpy()
 
 
-def shopee_dataset(download_dir):
+def path_expander(path, base_folder):
+    path_l = path.split(";")
+    return ";".join([os.path.abspath(os.path.join(base_folder, path)) for path in path_l])
+
+
+def _read_byte(file):
+    with open(file, "rb") as image:
+        f = image.read()
+        b = bytearray(f)
+    return b
+
+
+def path_to_bytearray_expander(path, base_folder):
+    path_l = path.split(";")
+    return [_read_byte(os.path.abspath(os.path.join(base_folder, path))) for path in path_l]
+
+
+def shopee_dataset(
```

Comment:
Besides unit tests, where do we use this dataset? It doesn't fit the context of this file.

Reply:
It is used in a few tutorials at the moment. Do you think creating a `datasets.py` under the `./utils` will help?

Reply:
We usually put the data preparation logic inside the tutorials, e.g.,  https://auto.gluon.ai/stable/tutorials/multimodal/multimodal_prediction/beginner_multimodal.html.